app/views/sessions/start.html.erb

```diff
@@ -36,6 +36,12 @@
                                   errors: @handler_result.try(:errors),
                                   error_field_classes: "alert alert-danger") %>
 
+      <% if @client_app  %>
+          <% if @client_app.name.downcase.include? "tutor" %>
```

Comment:
Could avoid the double `if` by using `&&` or `try!()` but it's very minor.

Reply:
I actually cringed writing this - but it was erroring out without it. I didn't know about the `&&`, let me update this.